apax/md/nvt.py

```diff
@@ -19,12 +19,36 @@
 from apax.md.io import H5TrajHandler
 from apax.md.md_checkpoint import load_md_state
 from apax.model import ModelBuilder
-from apax.train.eval import load_params
+from apax.train.checkpoints import restore_parameters
 from apax.utils import jax_md_reduced
 
 log = logging.getLogger(__name__)
 
 
+def create_energy_fn(model, params, numbers, box, n_models):
+    def ensemble(params, R, Z, neighbor, box, offsets):
+        vmodel = jax.vmap(model, (0, None, None, None, None, None), 0)
+        energies = vmodel(params, R, Z, neighbor, box, offsets)
+        energy = jnp.mean(energies)
+
+        return energy
+
+    if n_models > 1:
+        energy_fn = ensemble
+    else:
+        energy_fn = model
+
+    energy_fn = partial(
+        energy_fn,
+        params,
+        Z=numbers,
+        box=box,  # TODO IS THIS CORRECT FOR NPT???
```

Comment:
> # TODO IS THIS CORRECT FOR NPT???

I don't know, but if you want to merge this, one should certainly raise an issue that this must be checked!